jetty-core/jetty-http/src/test/java/org/eclipse/jetty/http/MultiPartFormDataTest.java

```diff
@@ -677,7 +674,7 @@ public void testPartWithBackSlashInFileName() throws Exception
 
         String contents = """
             --AaB03x\r
-            content-disposition: form-data; name="stuff"; filename="Taken on Aug 22 \\ 2012.jpg"\r
+            content-disposition: form-data; name="stuff"; filename="Taken on Aug 22 \\\\ 2012.jpg"\r
```

Comment:
This feels weird. Why there are now 2 backslashes?

Reply:
Because we used to forgive browsers for not properly escaping "\" in filenames.  Now we are going to insist they quote them correctly.    

Could be a bad idea.   @joakime can you do some tests on windows to see how "\" is treated by current browsers?

Reply:
Reverted.   But there is no good solution.  See comments below.